2001-3000/LIT2365Storyo.xml

```diff
@@ -74,7 +66,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: text record</change>
          <change who="MKr" when="2024-09-20">Added relations, witnesses, incipit</change>
-         <change who="NV" when="2024-11-04">Added an abstract</change>
+         <change who="NV" when="2024-11-04">Added an abstract, bibl, deleted list of witnesses since the text is unpublished.</change>
```

Comment:
I do not understand the argument. Is it not certain or incorrect, is there any copyright issue or why should the reference be deleted?